### ISSUE TITLE
Batch norm on trained generator issue

### ISSUE THREAD

**Issue by kurtisdavid** [csinva/gan-pretrained-pytorch#2]
Hello,

I am currently using your implementation for a project, and found a small detail that should be put in the notebook -- you need to call G.eval() if using the pretrained weights, since it contains batch norm layers. I found that if this isn't done, results would differ across batches vs. individual outputs.

**Comment by csinva** [comment]
Thanks for the catch - you're absolutely right! I've fixed this in the notebooks now.
